indication_paths.yaml

```diff
@@ -13659,3 +13659,2035 @@
         label: Disease
         name: Allergic rhinitis
     reference: https://go.drugbank.com/drugs/DB00835#mechanismofaction
+- directed: true
+  graph:
+    disease: Allergic rhinitis
+    disease_mesh: MESH:D065631
+    drug: mizolastine
+    drug_mesh: MESH:C076170
+    drugbank: DB12523
```

Comment:
Just realised I have been leaving the 'DB:` out. Sorry for that. Will add it to the forthcoming commits/pull requests.

Reply:
Not super important... I have code in place to automatically add them in when missing.